config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml

```diff
@@ -149,6 +149,9 @@ presubmits:
         args:
           - runner.sh
           - "./scripts/ci-e2e.sh"
+        env:
+          - name: GINKGO_SKIP
+            value: "\\[PR-Blocking\\] \\[K8s-Upgrade\\]"
```

Comment:
why are we skipping PR-Blocking in presubmit? seems counterintuitive

Reply:
might be worth adding a comment
we are skipping PR-Blocking because the are already run as part of pull-cluster-api-e2e-full, and we are skipping K8s-Upgrade because we are running separate test for KCP upgrade, MP upgrades, MD upgrades, Conformance

Reply:
Added a comment to explain clearly.